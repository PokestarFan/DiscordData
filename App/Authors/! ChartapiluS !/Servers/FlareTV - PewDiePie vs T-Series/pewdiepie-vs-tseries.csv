ID,Content Length,Content
558989902629437442,21,like a alien speaking
558990353827627029,24,biar blajar english h3h3
558991108256956441,45,can you go to heaven from say "sub to pewd"??
558991748622319626,6,hiyaaa
558991892927479829,36,weebs detected <@407076961786134539>
558992074595369012,24,B A C O T     A N Y I NG
558993195791548429,25,but why the subs so many?
558993211767652361,9,like pewd
558993284425318413,20,its why i hate t-gay
558994940743385088,29,Ice cream = I scream = ScreaM
558995657680224257,38,DMC 5 has great graphic when battle XD
558995709081419797,18,but i got lag lmao
558996661418852372,13,what time is?
558996952490967070,33,<@508486121798238209> RIP english
558997010368167948,29,dont say english when u can't
558997106996543488,21,its indonesian people
558997127016218658,4,h3h3
558997230183514112,29,T-series has left the server*
558997313968668672,44,<@558992002323316737> no one bullying u lmao
