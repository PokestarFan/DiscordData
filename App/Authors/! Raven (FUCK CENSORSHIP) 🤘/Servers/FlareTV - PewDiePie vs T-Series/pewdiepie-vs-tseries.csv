ID,Content Length,Content
558451776542736414,20,Weve hit 30k finally
