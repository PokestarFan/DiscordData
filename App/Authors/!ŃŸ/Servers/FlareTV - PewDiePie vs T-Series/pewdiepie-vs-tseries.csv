ID,Content Length,Content
559060292290478082,31,?mute <@558468497718640640> 100
559060302721843200,5,Muted
