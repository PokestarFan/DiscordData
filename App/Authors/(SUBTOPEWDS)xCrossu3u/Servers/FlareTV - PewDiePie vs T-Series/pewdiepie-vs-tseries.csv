ID,Content Length,Content
558849139874594817,62,Ooo our king is back on his throne_ 43 thousand sub difference
558849432620367874,11,yes you are
558849457417093122,17,Pewds is the king
559046010924433439,54,What do you think_ who is going to win? I bet on Pewds
559046261018198038,3,Lol
559046316458639384,23,~~Moto Moto likes you~~
559049397397356575,21,Omae wa mou shindeiru
559050459617296425,60,Its not even true_ they are lying to make Pewdiepie look bad
559057032146190356,39,I hate how Youtube is on T series' side
559057171564986409,10,Dont spamm
