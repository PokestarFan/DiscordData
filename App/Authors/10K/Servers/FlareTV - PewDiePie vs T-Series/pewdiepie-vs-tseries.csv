ID,Content Length,Content
558443479173300237,0,
