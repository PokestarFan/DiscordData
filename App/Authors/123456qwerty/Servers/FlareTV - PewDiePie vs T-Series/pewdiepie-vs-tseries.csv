ID,Content Length,Content
559021931974623232,20,#ThePowerOfPewDiePie
