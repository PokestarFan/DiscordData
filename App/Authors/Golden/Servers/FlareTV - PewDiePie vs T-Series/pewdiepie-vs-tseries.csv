ID,Content Length,Content
560654206491688980,32,They are fucking 1000 subs ahead
560654256739450900,7,And how
560654611191431210,22,Join public lobby vc 2
